bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -755,6 +771,96 @@ def shutdown(self, should_dock: bool = False) -> None:
         finally:
             self.power_off()
 
+    def get_hand_image(self, is_rgb=True):
```

Comment:
why not use the images from the ROS node?

Reply:
We need unscaled un-processed images so rectification and camera intrinsics make sense.
That being said, we can re-use `get_image_responses` directly, but this I feel is a tad bit cleaner & better.